packages/scaffolding-cli/src/__tests__/domain/ssrGkeJenkins.bootstrap-config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "projectName": "test-app-ssr-gke",
+  "projectType": "ssr",
+  "platform": "gke",
+  "deployment": "jenkins",
+  "advancedConfig": true,
+  "createConfig": true,
+  "cloudRegion": "uksouth",
+  "cloudResourceGroup": "string",
+  "businessCompany": "comp",
+  "businessProject": "biz",
+  "businessDomain": "domng",
+  "businessComponent": "component56",
+  "sourceControlRepoType": "github",
+  "sourceControlRepoName": "my-repo-git",
+  "sourceControlRepoUrl": "https://github/sample.git",
+  "terraformBackendStorage": "azure_blob",
```

Comment:
might be worth just changing this to storage?

Reply:
you mean GCS?